pkgs/development/coq-modules/ssprove/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, mkCoqDerivation, coq, version ? null
+, equations
+, mathcomp-ssreflect
+, mathcomp-analysis
+, extructures
+, deriving
+}:
+
+(mkCoqDerivation {
+  pname = "ssprove";
+  owner = "SSProve";
+
+  inherit version;
+  defaultVersion = with lib.versions; lib.switch [coq.coq-version mathcomp-ssreflect.version] [
+    { cases = ["8.18" (range "2.1.0" "2.2.0")]; out = "0.2.0"; }
```

Comment:
You may have removed the case for Coq 8.19 by mistake.

Reply:
I did.
Thanks for paying attention!